.rubocop.yml

```diff
@@ -5,9 +5,42 @@ AllCops:
     - 'lib/rubygems/resolver/molinillo/**/*'
   TargetRubyVersion: 2.2
 
+Layout/AccessModifierIndentation:
+  Enabled: true
+
+Layout/BlockAlignment:
+  Enabled: true
+
+Layout/CaseIndentation:
+  Enabled: true
+
+Layout/ClosingParenthesisIndentation:
+  Enabled: true
+
 Layout/CommentIndentation:
   Enabled: true
 
+Layout/ElseAlignment:
+  Enabled: true
+
+# Force Unix line endings.
+Layout/EndOfLine:
+  Enabled: true
+  EnforcedStyle: lf
+
+Layout/IndentHash:
+  Enabled: true
+  EnforcedStyle: consistent
+
+# NOTE: When used with --auto-correct, this only seems to auto-correct the
+#       first array item for some reason.
```

Comment:
Yeah, I noticed this when I reviewed the other PR. See https://github.com/rubygems/rubygems/pull/2407#discussion_r221091861. Given this limitation, I don't think it makes sense to use `Layout/IndentArray` but not `Layout/AlignArray`, so could we enable `Layout/AlignArray` as well?

Reply:
I thought there was something like that, but couldn't find it. Thanks for the info!

Reply:
Attempting to enable AlignArray revealed a potential bug, or at the very least some questionable code lol https://github.com/rubygems/rubygems/issues/2437